identity/client/index.html

```diff
@@ -18,5 +18,10 @@
   <body>
     <div id="root"></div>
     <script type="module" src="/src/main.tsx"></script>
+    <script th:inline="javascript" type="text/javascript" th:nonce="${clientConfigNonce}">
+      window.clientConfig = {
+        "VITE_IS_OIDC": "[[${isOidc}]]"
```

Comment:
❓ Do you think we can make the template more general? 

So that in the future we don't have to modify it if we need to add more `clientConfig` properties from the backend.

Reply:
I had trialed something like that, using loops but every instance I found was to do with tables. I'll have another look quickly though

Reply:
I saw in other webapps, they have an endpoint `GET /clientConfig.js` that would return the following string
```
window.clientConfig = { isOIDC: true }
```
Then the endpoint is added to the html as an src to a script tag.

See example [here](https://github.com/camunda/camunda/blob/75a83350b0c92bba57a2b522eb84dc5751d1c5ff/tasklist/client/index.prod.html#L7) for tasklist 

Would something like this work for identity?

Reply:
We could do that but it would need some more changes I think (FE and BE) so for now I would keep with this approach to ensure we have the functionality in place for alpha5 and alter as necessary after. 

I've now updated the code here to be easily extended by just passing a map into the templates.